adal/src/main/java/com/microsoft/aad/adal/TokenCacheAccessor.java

```diff
@@ -80,33 +86,60 @@ TokenCacheItem getATFromCache(final String resource, final String clientId, fina
         
         return accessTokenItem;
     }
-    
+
     /**
      * @return {@link TokenCacheItem} for regular token cache entry.  
      */
     TokenCacheItem getRegularRefreshTokenCacheItem(final String resource, final String clientId, final String user) {
+        final CacheEvent cacheEvent = startCacheTelemetryRequest(EventStrings.TOKEN_TYPE_RT);
```

Comment:
RT lookup, found=true, http_event, api_successful in best cases.

---
In reply to: [90368738](https://github.com/AzureAD/azure-activedirectory-library-for-android/pull/736#discussion_r90368738) [](ancestors = 90368738)